### DISCUSSION TITLE
is there any sql function available to decode term_to_binary?

### DISCUSSION THREAD

**Discussion by muhun** [emqx/emqx#8543] [Q&A]
I use emqtt to publish message , with term_to_binary(payload), but its hard to convert it back to payload in javascript client , is there any function available in emqx sql to convert it back to payload , like bin2hexstr ?

**Comment by lafirest** [comment]
There is no reason to use `term_to_binary` when you wanna communicate with a JS program, make sure your payload is in JSON format
